app/configurator/components/chart-type-selector.tsx

```diff
@@ -27,9 +20,10 @@ import { IconButton } from "@/configurator/components/icon-button";
 import { useChartType } from "@/configurator/config-form";
 import { ConfiguratorStateWithChartConfigs } from "@/configurator/configurator-state";
 import { useDataCubesComponentsQuery } from "@/graphql/hooks";
-import { Icon } from "@/icons";
 import { useLocale } from "@/locales/use-locale";
 
+import { InfoIconTooltip } from "./info-icon-tooltip";
```

Comment:
Super-nit: I think the default behavior of VS Code is to use relative imports up to 2 levels high, and aliased is available for longer ones. I added this setting to VS Code settings to always use aliased one:

```
"typescript.preferences.importModuleSpecifier": "non-relative",
```

maybe it could be added to workspace settings so it's shared?

Reply:
I already have it 🤷 I just noticed that info-icon-tooltip should be in app/components directly, not in configurator/, I am changing this.